### DISCUSSION TITLE
使用useCanvas2D进行渲染导致动画显示不全，缺少一块儿

### DISCUSSION THREAD

**Discussion by ahsdhajshqwje** [Tencent/libpag#1652] [General]
const PAG = await window.libpag.PAGInit();
      let url = "/pag/" + this.url + ".pag"
      let id = this.renderingId
      const buffer = await fetch(url).then((response) => {
        return response.arrayBuffer()
      })

      const pagFile = await PAG.PAGFile.load(buffer)

   const pagComposition = PAG.PAGComposition.make(pagFile.width(), pagFile.height());

    pagComposition.addLayer(pagFile);
      const canvas = document.getElementById(id);
      if (canvas !== null) {
        canvas.width = pagComposition.width();
        canvas.height = pagComposition.height();

        const pagView = await PAG.PAGView.init(pagComposition, canvas, { useCanvas2D: true });

        pagView.setRepeatCount(this.playNum);
        await pagView.play();

      }

**Comment by zhalice2011** [comment]
你好, 请问方便提供具体的 PAG 文件吗
